src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <div>
+    <p>
+      Add the people and corporations/firms who will have a role in your company. People can have multiple roles;
+      Corporation/firms can only be Incorporators.
```

Comment:
Mockup shows "Corporation" as lower case in this description. Please confirm with Scott.

Reply:
Will follow up with Scott! 👍